docs/user-guide/node-selection/pod-with-pod-affinity.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: with-newlabels
+  annotations:
+    scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
The validation verifies that the syntax is ok, not that it can be met.  I'll open an issue around an analysis tool.